app/assets/stylesheets/shared/tables.scss

```diff
@@ -107,30 +71,34 @@
   }
 }
 
+.name {
+  width: 100px;
+}
+
+.email {
+  width: 250px;
+}
+
+.actions {
+  width: 100px;
+}
+
 .decision-button {
+  background: $medium-gray;
+  border-radius: 6px;
```

Comment:
Properties should be ordered background, border, border-radius, color, display, font-size, font-weight, line-height, margin, padding, text-decoration